network-area-diagram/src/main/java/com/powsybl/nad/svg/metadata/DiagramMetadata.java

```diff
@@ -107,7 +108,8 @@ public DiagramMetadata addMetadata(Graph graph) {
                 getPrefixedId(node.getDiagramId()),
                 node.getEquipmentId(),
                 round(node.getX()),
-                round(node.getY()))));
+                round(node.getY()),
+                node instanceof VoltageLevelNode vlNode && vlNode.isFictitious())));
```

Comment:
Could you move fictitious to Node interface and AbstractNode class?
```suggestion
                node.isFictitious())));
```

Reply:
method isFictitious moved to Node interface